### PR TITLE
Implement component  cart product 

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component  cart product 

image

image

storybook:  https://6684faab2d0c99a3e93bf145-fppsufnqrj.chromatic.com/

### CODE REVIEW

File: src/components/Icon/BookMarkIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+interface BookMarkIconProps {
+  onChange?: () => void;
+}
```
Comment (hungdo-agilityio) [inline src/components/Icon/BookMarkIcon.tsx]: Icon should not have this prop, this should belong to parent element

Comment (thanhtrantruong47) [reply src/components/Icon/BookMarkIcon.tsx]: Fixed

File: src/components/ProductCard/ProductCart.tsx [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+  ratingNumbers,
+  onChange,
+}: ProductCardProps) => (
+  <div className={styles.product}>
+    <div className={`${styles.imageGroup} ${utils.relative}`}>
+      <img className={styles.productImage} src={image} alt={title} />
+      <div className={styles.bookMark}>
+        <div onChange={onChange}>
+          <BookMarkIcon />
+        </div>
```
Comment (hungdo-agilityio) [inline src/components/ProductCard/ProductCart.tsx]:
```
<button aria-label="toggle bookmark" onChange={onChange}>
  <BookMarkIcon />
</button>
```

Comment (thanhtrantruong47) [reply src/components/ProductCard/ProductCart.tsx]: Fixed

**Review by hungdo-agilityio — APPROVED** [review]

**Pull request merged**